gctscript/modules/ta/indicators/indicators_test.go

```diff
@@ -10,12 +10,15 @@ import (
 
 	objects "github.com/d5/tengo/v2"
 	"github.com/thrasher-corp/go-talib/indicators"
+	"github.com/thrasher-corp/gocryptotrader/gctscript/wrappers/validator"
 )
 
 const errFailedConversion = "0 failed conversion"
 
 var (
-	ohlcvData = &objects.Array{}
+	ohlcvData        = &objects.Array{}
+	ohlcvDataInvalid = &objects.Array{}
+	testString       = "1D01TH0R53"
```

Comment:
> I'm happy with the way the code looks here, but I'm still requesting changes because I still cannot match your script examples to the output in BTC Markets. I've tried running `go get -u` just in case I've got stuck.
> 
> I've documented my attempts at trying to match output. If I am missing anything, then please let me know so I can test this better.
> 
> The values I am matching against are the _latest_ displayed on the BTC markets charts on their website versus the _latest_ result in the script output
> 

I have pushed an update to the example scripts as  the moing average ones were previous 100 day 1 hour candles that dont have a matching range on BTC markets

but going over them all again now and i have messaged you on slack with some more details